documentation/jetty-documentation/pom.xml

```diff
@@ -70,6 +70,7 @@
               <goal>process-asciidoc</goal>
             </goals>
             <configuration>
+              <skip>${skipTests}</skip>
```

Comment:
to avoid repeating this, you should try add a single entry in the `<properties>` section such:
```
<properties>
  <asciidoctor.skip>${skipTests}</asciidoctor.skip>
</properties>
```